src/Flash.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const BOOL_PROPS_TO_CLASS_NAMES = {
+  info: ['rev-Flash--info'],
+  success: ['rev-Flash--success'],
+  warning: ['rev-Flash--warning'],
+  alert: ['rev-Flash--alert'],
+  persistent: ['rev-Flash--persistent'],
+  fade: ['rev-Flash--fade'],
+  dismissible: ['rev-Flash--dismissible'],
```

Comment:
we can keep this prop for now but delete all other mentions of it. We'll make an enhancement ticket to add the dismissible flash

Reply:
done